dateparser/conf.py

```diff
@@ -31,19 +31,21 @@ class Settings:
     _pyfile_data = None
     _mod_settings = dict()
 
-    def __init__(self, settings=None):
-        if settings:
-            self._updateall(settings.items())
-        else:
+    def __init__(self, **kwargs):
```

Comment:
@Gallaecio @alex-triphonov 
would it be possible to keep the `settings=None` and, in case it's used, raise a deprecation warning and preserve the old behavior? 

Something like:
```
def __init__(self, settings=None, **kwargs):
    if settings:
        # raise exception (deprecation warning, not concurrent) + old behavior
    else:
        # new behavior using kwargs
```

I know the code won't be nice, but I think in that way we could keep the backward compatibility until the next major version.

(You should do the same for `get_key()` and maybe something similar in the other two methods you touched).

Maybe I'm saying something stupid, sorry guys, I didn't have too much time to play with the code.

Reply:
No this is not possible- settings are still being used, but in kwargs for more convenience, the only way to separate  behaviours is to add some new _kwarg_ like `new_settings` but is old behaviour with fewer `Settings` instances that much essential?

Reply:
To me it’s mostly a matter of API. The current change would require to postpone this to dateparser 2.x, as it is backward compatible. If we can figure a backward compatible implementation, this can go on the next dateparser version.